src/js/jsx/search/SearchBar.jsx

```diff
@@ -458,13 +475,16 @@ define(function (require, exports, module) {
 
         /**
          * @private
-         * @type {Datalist~onKeyDown}
+         * @param {SyntheticEvent} event
+         * @param {string} selectedID the id of the selected item in the datalist
          */
         _handleKeyDown: function (event, selectedID) {
             switch (event.key) {
-                case "Return":
                 case "Enter":
-                    if (selectedID === PLACEHOLDER_ID) {
+                    // The only time selectedID is falsey is when an Adobe Stock search
```

Comment:
Presumably you are building infrastructure that is not specific to Adobe Stock, so this comment should not be specific to Adobe Stock.